Assets/HoloToolkit-Examples/Input/Scenes/MotionControllerTest.unity

```diff
@@ -7091,7 +7091,34 @@ Prefab:
       objectReference: {fileID: 0}
     - target: {fileID: 114710303647202208, guid: d29bc40b7f3df26479d6a0aac211c355,
         type: 2}
-      propertyPath: LeftControllerOverride
+      propertyPath: AlternateLeftControllerModel
+      value: 
+      objectReference: {fileID: 1097318689274706, guid: 869d340c69dbcef459070c1a159d38ac,
+        type: 2}
+    - target: {fileID: 114710303647202208, guid: d29bc40b7f3df26479d6a0aac211c355,
+        type: 2}
+      propertyPath: AlternateRightControllerModel
+      value: 
+      objectReference: {fileID: 1097318689274706, guid: 869d340c69dbcef459070c1a159d38ac,
+        type: 2}
+    - target: {fileID: 1493965172262900, guid: d29bc40b7f3df26479d6a0aac211c355, type: 2}
+      propertyPath: m_IsActive
+      value: 1
+      objectReference: {fileID: 0}
+    - target: {fileID: 114710303647202208, guid: d29bc40b7f3df26479d6a0aac211c355,
+        type: 2}
+      propertyPath: AlwaysUseAlternateModel
```

Comment:
I think there might be some stale references in here, if you could re-save the scene and update it please, that'd be great!

Reply:
Hmm, yeah it doesn't look like Unity is removing the old stuff. Cleaning up now.